faq.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: home
 title: FAQ
-nav_order: 7
+nav_order: 8
 nav_titles: true
 titles_max_depth: 2
 ---
```

Comment:
(Not related to this line)
Could you add a link to the new iOS (or macOS, see above) page in the "_What are the supported operating systems?_" section please?, e.g.

    It is most straightforward when applications use the system’s [frameworks](https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/improving_network_reliability_using_multipath_tcp), as explaining on the [macOS](/macOS.html) page.